src/test/java/com/evuv/builders/ExpressionBuilderTest.java

```diff
@@ -195,7 +196,41 @@ public void testNotFalse() throws EventBindingException {
 	}
 	
 	
-	
+	@Test
+	public void testInExpression() throws EventBindingException {
```

Comment:
can you please also add 2 more tests form a json ?

Reply:
Done